### ISSUE TITLE
Can't build with gradle 6+

### ISSUE THREAD

**Issue by dzelemba** [GoogleCloudPlatform/healthcare-dicom-dicomweb-adapter#74]
```
gradle build
Starting a Gradle Daemon (subsequent builds will be faster)

FAILURE: Build failed with an exception.

* Where:
Build file '/Users/dzelemba/code/healthcare-dicom-dicomweb-adapter/export/build.gradle' line: 81

* What went wrong:
A problem occurred evaluating project ':export'.
> Could not create task ':export:downloadLicenses'.
   > Replacing an existing task that may have already been used by other plugins is not supported.  Use a different name for this task ('downloadLicenses')
```

**Comment by fernandogbi** [comment]
I got it using gradle 5.6, try to install this version and run de build

**Comment by fernandogbi** [comment]
> ```
> gradle build
> Starting a Gradle Daemon (subsequent builds will be faster)
> 
> FAILURE: Build failed with an exception.
> 
> * Where:
> Build file '/Users/dzelemba/code/healthcare-dicom-dicomweb-adapter/export/build.gradle' line: 81
> 
> * What went wrong:
> A problem occurred evaluating project ':export'.
> > Could not create task ':export:downloadLicenses'.
>    > Replacing an existing task that may have already been used by other plugins is not supported.  Use a different name for this task ('downloadLicenses')
> ```

I fix this issue using gradle version 5.6, try to install this version and run the build again

**Comment by dzelemba** [comment]
Thanks! Yup using Gradle version 5.6 works for me, I'll keep this open to track fixing for Gradle v6+.
